RHEL/7/input/oval/grub2_enable_fips_mode.xml

```diff
@@ -0,0 +1,50 @@
+<def-group>
+  <definition class="compliance" id="grub2_enable_fips_mode" version="1">
+    <metadata>
+      <title>Enable FIPS Mode in GRUB2</title>
+      <affected family="unix">
+        <platform>Red Hat Enterprise Linux 7</platform>
+      </affected>
+      <description>Look for argument fips=1 in the kernel line in /etc/default/grub.</description>
+      <reference source="galford" ref_id="20160608" ref_url="test_attestation" />
+    </metadata>
+    <criteria operator="AND">
+      <extend_definition comment="Installed OS is RHEL7" definition_ref="installed_OS_is_rhel7" />
+      <criteria operator="OR">
```

Comment:
We might have to detect if the GRUB_CMDLINE_LINUX_DEFAULT line exists then use the "AND" operand otherwise use the "OR" operand. I will look into fixing this tomorrow.